### PR TITLE
Php examples

### PR DESCRIPTION
**Pull request by ryannguyen16**
Adds PHP examples.

### CODE REVIEW

**Review by jchen293 — CHANGES_REQUESTED** [review]
Please remove the `venv` folder, this isn't part of this PR

**Review by Justintime50 — APPROVED** [review]

File: official/docs/php/addresses/CreateAddress.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "state"   => "CA",
+    "zip"     => "94104",
+    "country" => "US",
+    "company" => "EasyPost",
+    "phone"   => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create($addressParams);
+
+echo $addresss;
```
Comment (jchen293) [inline official/docs/php/addresses/CreateAddress.php:18]:
```suggestion
echo $address;
```
extra `s`

File: official/docs/php/addresses/CreateAndVerifyAddress.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "state"   => "CA",
+    "zip"     => "94104",
+    "country" => "US",
+    "company" => "EasyPost",
+    "phone"   => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create_and_verify($addressParams);
+
+echo $addresss;
```
Comment (jchen293) [inline official/docs/php/addresses/CreateAndVerifyAddress.php:18]:
```suggestion
echo $address;
```

File: official/docs/php/addresses/RetrieveAddressList.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");
+
+$addresses = \EasyPost\Address::all([
+    "page_size" => 5
+]);
+
+echo $addressses;
```
Comment (jchen293) [inline official/docs/php/addresses/RetrieveAddressList.php:9]:
```suggestion
echo $addresses;
```

File: official/docs/php/addresses/StrictVerifyAddress.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "state"          => "CA",
+    "zip"            => "94104",
+    "country"        => "US",
+    "company"        => "EasyPost",
+    "phone"          => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create($addressParams);
+
+echo $addressses;
```
Comment (jchen293) [inline official/docs/php/addresses/StrictVerifyAddress.php:20]:
```suggestion
echo $address;
```

File: official/docs/php/addresses/VerifyExistingAddress.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    "country" => "US",
+    "company" => "EasyPost",
+    "phone"   => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create($addressParams);
+
+$address->verify();
+
+echo $addressses;
```
Comment (jchen293) [inline official/docs/php/addresses/VerifyExistingAddress.php:20]:
```suggestion
echo $address;
```

File: official/docs/php/shipments/CreateShipment.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        'phone' => '3331114444',
+        'email' => 'support@easypost.com'
+    ],
+    "parcel" => [
+        "length" => 20.2,
+        "width" => 10.9,
+        "height" => 5,
+        "weight" => 65.9
+    ]
+]);
```
Comment (jchen293) [inline official/docs/php/shipments/CreateShipment.php:33]: missing print statement

**Review by jchen293 — CHANGES_REQUESTED** [review]

File: official/docs/php/addresses/RetrieveAddress.php [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");
+
+$address = \EasyPost\Address::retrieve("adr_...");
+
+echo $addresss;
```
Comment (jchen293) [inline official/docs/php/addresses/RetrieveAddress.php:7]:
```suggestion
echo $address;
```

My bad, here is another one

File: official/docs/php/addresses/StrictVerifyAddress.php [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    "state"          => "CA",
+    "zip"            => "94104",
+    "country"        => "US",
+    "company"        => "EasyPost",
+    "phone"          => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create($addressParams);
+
+echo $addresses;
```
Comment (jchen293) [inline official/docs/php/addresses/StrictVerifyAddress.php:20]:
```suggestion
echo $address;
```

File: official/docs/php/addresses/VerifyExistingAddress.php [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    "country" => "US",
+    "company" => "EasyPost",
+    "phone"   => "415-123-4567"
+];
+
+$address = \EasyPost\Address::create($addressParams);
+
+$address->verify();
+
+echo $addresses;
```
Comment (jchen293) [inline official/docs/php/addresses/VerifyExistingAddress.php:20]:
```suggestion
echo $address;
```

File: official/docs/php/api_keys/RetrieveApiKey.php [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (jchen293) [inline official/docs/php/api_keys/RetrieveApiKey.php:1]: This file should be plural `RetrieveApiKeys.php`

File: official/docs/php/shipments/InsureShipment.php [hunk 11]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");
+
+$purchasedShipment = \EasyPost\Shipment::retrieve("shp_...");
+
+$purchasedShipment->insure(['amount' => 100]);
+
+echo $shipment;
```
Comment (jchen293) [inline official/docs/php/shipments/InsureShipment.php:9]:
```suggestion
echo $purchasedShipment;
```

**Review by jchen293 — CHANGES_REQUESTED** [review]
I noticed that we used `double quotes` and `single quotes` in some files. I see we use both `double quotes` and `single quote` in some associated arrays, and also when retrieving an object by passing either `'obj_...'` or `"obj_..."` in Object::retrieve().

**Comment by Justintime50** [comment]
@jchen293 great callout. I added a rule to the PHP linter to check for double vs single quotes and fixed all occurances.

**Review by nwithan8 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]

**Pull request merged**